src/platforms/linux/linuxcontroller.cpp

```diff
@@ -168,6 +120,7 @@ void LinuxController::operationCompleted(QDBusPendingCallWatcher* call) {
   emit disconnected();
 }
 
+#if 0
```

Comment:
Should this be removed?

Reply:
Yes, it should. I must have forgotten to clean this up after it was no longer necessary.